pkgs/servers/miniflux/default.nix

```diff
@@ -1,17 +1,18 @@
-{ stdenv, buildGoPackage, fetchFromGitHub, installShellFiles }:
+{ stdenv, buildGoPackage, fetchFromGitHub, installShellFiles, nixosTests }:
 
 buildGoPackage rec {
```

Comment:
Did you consider also migrating this to the new `buildGoModule` function?
The project seems to have `go.mod` and `go.sum` at https://github.com/miniflux/miniflux/tree/2.0.22

Reply:
Yeah, I needed to use the `go.mod` to begin with to get it to build at all using vgo2nix.

But I've pushed a fixup to migrate to `buildGoModule`, I can squash them if you think the changes are reasonable :)